cmd/scaffold.go

```diff
@@ -526,8 +532,20 @@ func (fnb *FlowNodeBuilder) initState() {
 	// 2) the node id is a new one for a new spork, but the bootstrap data has not been updated.
 	myID, err := flow.HexStringToIdentifier(fnb.BaseConfig.nodeIDHex)
 	fnb.MustNot(err).Msg("could not parse node identifier")
-	self, err := fnb.State.Final().Identity(myID)
-	fnb.MustNot(err).Msgf("node identity not found in the identity list of the finalized state: %v", myID)
+
+	var self *flow.Identity
+	if fnb.unstaked {
```

Comment:
In the case of the unstaked node, no attempts to check if the the node exist in the protocol data should be made.